django/contrib/sessions/backends/signed_cookies.py

```diff
@@ -13,9 +13,9 @@ def load(self):
             return signing.loads(
                 self.session_key,
                 serializer=self.serializer,
-                # This doesn't handle non-default expiry dates, see #19201
```

Comment:
Can we remove this comment? Does it actually solves ticket-19201 as well? :thinking:  I think we need more tests to actually confirm this.

Reply:
This doesn't solve #19201, however the comment is no longer valid as signing now handles non-default expiry dates.